apps/web/src/index.css

```
@@ -115,145 +122,94 @@
   }
 }
 
+/* ── Terminal Dark palette ─────────────────────────────────────────── */
 :root {
-  --radius: 0.625rem;
+  --radius: 6px;
   color-scheme: dark;
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)